app/src/db/DatabaseCon.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Src\db;
+
+class DatabaseCon
+{
+    private $dbConnection = null;
+
+    public function __construct()
+    {
+
+    }
+
+    public function getConnection()
+    {
+        $host = getenv('DB_HOST');
+        $port = getenv('DB_PORT');
+        $db = getenv('DB_DATABASE');
+        $user = getenv('DB_USERNAME');
+        $pass = getenv('DB_PASSWORD');
+
+        try {
+            $this->dbConnection = new \PDO(
```

Comment:
@jjsimonAB specify PDO on top to avoid having to use \PDO

Also use https://phpenthusiast.com/blog/the-singleton-design-pattern-in-php to avoid opening a connection for each query